### PR TITLE
Record in tracks merchants returning from the kyc reminder email

### LINKED ISSUE #3957: Record in Tracks merchants returning from the KYC reminder email
### Description
Track merchants visiting the Connect page with the special URL param to redirect them to the KYC flow.

This URL is shown in the onboarding reminder emails we sent to the merchants. This issue is for recording this in Tracks for further analysis.

### Acceptance criteria
* Visiting `{website url}/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect&wcpay-connect-redirect={ email_type }` must record a `wcadmin_wcpay_kyc_reminder_merchant_returned` event in Tracks
* The props of the event must be the following ones according to the `wcpay-connect-redirect` value:
initial: [ type: 'initial', 'follow_number': '0' ]
1: [ type: 'follow', 'follow_number': '1' ]
2: [ type: 'follow', 'follow_number': '2' ]
3: [ type: 'follow', 'follow_number': '3' ]
4: [ type: 'follow', 'follow_number': '4' ]

### Testing instructions
Visiting the URL mentioned above must trigger the mentioned event with the expected properties for each value of `wcpay-connect-redirect`.

### Additional context
From the project paJDYF-36c-p2

### PR DESCRIPTION
**Pull request by a-danae**
Fixes #3957

#### Changes proposed in this Pull Request

Record in Tracks the visit to the URL for redirecting merchants to the KYC flow.

The parameter `wcpay-connect-redirect`  in the redirect URL will be set in the onboarding reminder email. Merchants. would visit the URL we're tracking here by clicking this link in that email.

#### Testing instructions
* Make sure Tracking is enabled in WC options. That's in `{your site}/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`.
* Make your site publicly accessible. `npm run tube:start` would do the trick if your're using Jurassic Tube. I don't recall if this was required but mentioning it just in case

Initial:
* Visit `{website url}/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect&wcpay-connect-redirect=initial` 
* Go to Live Tracks and search `wcadmin_wcpay_kyc_reminder_merchant_returned`
* Wait ~5 minutes after visiting the URL (it could take ~10 mins for the event to show up) 
* Confirm the event has `eventprops [ type: initial, follow: 0 ]` in its properties

Follow-ups
* Visit `{website url}/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect&wcpay-connect-redirect={ 1 | 2 | 3 | 4}` 
* Go to Live Tracks and search `wcadmin_wcpay_kyc_reminder_merchant_returned`
* Wait ~5 minutes after visiting the URL (it could take ~10 mins for the event to show up) 
* Confirm the event has `eventprops [ type: follow, follow: 1 | 2 | 3 | 4 ]` in its properties

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) : QA Testing Not Applicable

### CODE REVIEW

**Comment by leonardola** [comment]
Looks good to me. Let's wait for @alefesouza review before merging

**Review by alefesouza — APPROVED** [review]
LGTM 🚢

**Pull request merged**
